apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/[resourceType]/[resource]/index.vue

```diff
@@ -0,0 +1,3 @@
+<template>
+  <TableRowDetails />
```

Comment:
suggest to look in to using the routes.ts if not using the pages feature

Reply:
I think if we consider 'all' as a special filter set that we can move TableRowDetails back to be a page.